keras_cv/layers/preprocessing/random_shear_test.py

```diff
@@ -73,7 +74,7 @@ def test_return_shapes(self):
     def test_single_image_input(self):
         """test for single image input"""
         xs = tf.ones((512, 512, 3))
-        ys = tf.ones(shape=(5, 4))
+        ys = tf.ones(shape=(5, 5))
```

Comment:
So we always require users to call `add_class_id` before calling aug layers?

Reply:
if they don't have a class ID already, then yes!